specification/subscription/resource-manager/readme.md

```diff
@@ -57,6 +57,18 @@ title: SubscriptionClient
 description: The subscription client
 ```
 
+### Tag: package-2021-01-preview
+
+These settings apply only when `--tag=package-2021-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-01-preview'
+input-file:
+- Microsoft.Subscription/stable/2016-06-01/subscriptions.json
```

Comment:
The readme file doesn't refer to the new privatepreview version of the spec.  Also private previews are generally not checked in to the public repo

Reply:
Thanks for the response!

1. Missed the readme change. Updated the PR.
2. So most of the parts of this swagger were reviewed here - https://github.com/Azure/azure-rest-api-specs/pull/12551. However, there was a decision to make it privatepreview, so that we can make incremental changes and at the same time allow customers to use. Hence, this PR with the privatepreview version (and some updates to request and response bodies).
3. The reason for moving to a new version was that these changes are breaking when compared to the previous version. I have updated the readme to just have the privatepreview version as input. Looks like this line was added by mistake `- Microsoft.Subscription/stable/2016-06-01/subscriptions.json`